google/ggrc-core
# [WIP] GGRC-6349 Refactor Dropdown component

# Issue description

There are at least two types of dropdown element that has different representation: 
general dropdown and multi-select dropdown.

Also, current dropdown value binding doesn't work in CanJS 3, so we have to move to typical 2-way binding instead.

# Steps to test the changes

Check the entire app for the usage of dropdown component and verify it works as supposed. Mostly usages can be found in states and custom attributes of business objects.

Also, there's usage in the cycle task group object task. You should set the type of the task to Dropdown, add some options, save it. And then in the cycle task modal Dropdown component with desired options would appear.

Verify that styles and code used for dropdown and multi-select dropdown are at least similar.

# Solution description

Completely refactor dropdown component. Implement it as a div with input for current value and ul for all available options.

# Sanity checklist


# PR Review checklist


# TODO:

- [ ] Make new Dropdown as accessible, as native select (navigation with typing option name and navigation with arrow keys would be fine too)
